src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -93,6 +96,10 @@ public function newAction(Request $request)
             // See http://symfony.com/doc/current/book/controller.html#flash-messages
             $this->addFlash('success', 'post.created_successfully');
 
+            if ($form['createAndShow']->isClicked()) {
```

Comment:
Yes, I guess it works. My comment was more about: "Symfony uses OOP/classes/methods and "others" use arrays". That's why I think `$form->get('createAndShow')` is more appropriate for Symfony. But again, I know nothing about this.